File: php/01-introduction/introduction.php
<?php
class Person {
    public $name;
    public $age;
    
    public function introduceYourself()
    {
        echo "I'm " . $this->name . " and I'm " . $this->age . " years old.";
    }
}

// create object
$personObject = new Person();

// set properties
$personObject->name = "Seyed Reza Bazyar";
$personObject->age = 25;

$personObject->introduceYourself();

File: cpp/01-introduction/introduction.cpp
#include <string>
#include <iostream>
​
using namespace std;
​
class Person
{
public:
    string name;
    int age;
​
    void introduceYourself()
    {
        cout << "I'm " << name << " and I'm " << age << " years old.";
    }
};
​
int main()
{
    // create object
    Person personObject;
​
    // set properties
    personObject.name = "Seyed Reza Bazyar";
    personObject.age = 25;
​
    personObject.introduceYourself();
    return 0;
}